### PR TITLE
[docs] Add missing fields to package.json (closes #761)

### LINKED ISSUE #761: Add missing fields to package.json
E.g. repo, description, keywords, etc.

### PR DESCRIPTION
**Pull request by inikulin**
\cc @VasilyStrelyaev @AlexanderMoskovkin

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,11 +1,53 @@
 {
   "name": "testcafe",
+  "description": "Automated browser testing for modern web development stack.",
```
Comment (VasilyStrelyaev) [inline package.json]:
```
  "description": "Automated browser testing for the modern web development stack.",
```

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by inikulin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**